Jenkinsfile.nightly

```diff
@@ -18,18 +18,12 @@ pipeline {
 			steps {
 				sh '''
 				sudo service postgresql restart
-				cp test/data/config.json test/data/genesis_block.json .
 				'''
 			}
 		}
-		stage('Run integration tests without syncing') {
+		stage('Run integration tests with syncing only') {
 			steps {
-				sh 'SYNCING_DISABLED=true npm test -- mocha:extensive:integration'
-			}
-		}
-		stage('Run integration tests without broadcasting') {
-			steps {
-				sh 'TOTAL_PEERS=2 BROADCASTING_DISABLED=true npm test -- mocha:extensive:integration'
+				sh 'TOTAL_PEERS=2 BROADCASTING=false npm test -- mocha:stress:integration'
```

Comment:
Please introduce consistent `ENV` variable names. We have `BROADCASTING` and `SYNCING_DISABLED`, both names is not consistent. Afterwards specify both variables here to explicitly specify what is going to be tested. 

```
sh 'TOTAL_PEERS=2 BROADCASTING=false SYNCING=true npm test -- mocha:stress:integration'
```

Reply:
`SYNCING_DISABLED` doesn't exist. It is `SYNCING` but for now we have decided not to use it. By default both are true (enabled).